docs/badge.html

```diff
@@ -122,7 +122,116 @@ <h5 class="my-1">Components</h5>
+                  with avatar.
+                </p>
+
+                <p>
+                  Badge status includes online badge, offline badge, and number
+                  badge. You can add this badges at four positions, top left,
+                  top right, bottom right, and bottom left.
+                </p>
+
+                <div class="demo-container">
+                  <div class="badge-container">
+                    <div class="avatar avatar-lg">
+                      <img
+                        class="img-responsive img-round"
+                        src="../assets/avatar-girl.jpg"
+                        alt=""
+                      />
+                      <span class="badge badge-lg badge-online top-left"></span>
+                    </div>
+                  </div>
+
+                  <div class="badge-container">
+                    <div class="avatar avatar-md">
+                      <img
+                        class="img-responsive img-round"
+                        src="../assets/avatar-girl.jpg"
+                        alt=""
+                      />
+                      <span class="badge badge-md badge-num top-right">3</span>
+                    </div>
+                  </div>
+
+                  <div class="badge-container">
+                    <div class="avatar avatar-sm">
+                      <img
+                        class="img-responsive img-round"
+                        src="../assets/avatar-girl.jpg"
+                        alt=""
+                      />
+                      <span
+                        class="badge badge-sm badge-offline bottom-right bold-txt"
+                      ></span>
+                    </div>
+                  </div>
+
+                  <div class="badge-container">
+                    <div class="avatar avatar-xs">
+                      <img
+                        class="img-responsive img-round"
+                        src="../assets/avatar-girl.jpg"
+                        alt=""
+                      />
+                      <span
+                        class="badge badge-xm badge-online bottom-left"
+                      ></span>
+                    </div>
+                  </div>
+                </div>
+
+                <figure class="code-container">
+                  <code>
+                    <script src="https://gist.github.com/junaidshaikh-js/c3ee5b153c63740826bd308c81ead938.js"></script>
+                  </code>
+                </figure>
+              </section>
+
+              <section>
+                <h2 id="badge-on-icons">Badge on Icons</h2>
+
+                <p>
+                  Number badges on icons is commonly used pattern to display
+                  number counts of notification or any other countable thing.
+                </p>
+
+                <div class="demo-container">
+                  <button class="btn btn-icon btn-icon-lg badge-container">
```

Comment:
"btn-icon-lg" can be changed to "btn-lg" once you create button component

Reply:
Noted!